installation_tips/README.md

```diff
@@ -9,8 +9,6 @@ This environment will install:
  * spikeinterface-gui
  * phy
  * tridesclous
- * spyking-circus (not on mac)
- * herdingspikes (not on windows)
 
 Kilosort, Ironclust and HDSort are MATLAB based and need to be installed from source.
 Klusta does not work anymore with python3.8 you should create a similar environment with python3.6.
```

Comment:
3.6 is not supported. Do we still want this in beginner tips?

Reply:
oups no. I will clean also this